### DISCUSSION TITLE
Docker desktop failed to detect windows 10 21H2 virtual machine platform & WSL status

### DISCUSSION THREAD

**Discussion by tonymakt** [podman-desktop/podman-desktop#6397] [Q&A]
I got a company PC installed windows 10 LTSC ver.21H2, got HyperV and WSL installed and enabled.
But docker desktop failed to detect the status during pod man setup screen.
I had tried v1.5.3 & v.1.7.1 shown same issue
How to fix it?

**Comment by jeffmaury** [comment]
Which edition of Windows do you have ?

**Reply by tonymakt** [reply]
Windows 10 Enterprise LTSC
21H2
19044.3930
Windows Feature Experience Pack 1000.19053.1000.0
